app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/components/contextMenuEntry.tsx

```diff
@@ -9,13 +9,13 @@ import * as react from 'react'
 export interface ContextMenuEntryProps {
     disabled?: boolean
     title?: string
-    onClick: (event: react.MouseEvent<HTMLButtonElement>) => void
+    onClick: (event: React.MouseEvent<HTMLButtonElement>) => void
 }
 
 // This component MUST NOT use `useState` because it is not rendered directly.
 /** An item in a `ContextMenu`. */
-function ContextMenuEntry(props: react.PropsWithChildren<ContextMenuEntryProps>) {
-    const { children, disabled, title, onClick } = props
+function ContextMenuEntry(props: React.PropsWithChildren<ContextMenuEntryProps>) {
+    const { children, disabled = false, title, onClick } = props
```

Comment:
Why are we passing `disabled` as a prop if we're always setting it to `false`?

Reply:
this is a default value, not an override (otherwise, of course, it makes little sense to support in destructuring)